submissions/TaniaBondarenko/a-tiny-js-world-update/index.js

```diff
@@ -0,0 +1,73 @@
+class Inhabitant {
+  constructor(name, gender, saying, friends,legs) {
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+    this.legs = legs;
+  }
+  toString() {
+    return (` ${this.legs} legs. I\'m <b>${this.name}</b>, a ${this.gender}. To my friend ${this.friends} I always say: "${this.saying}"`);
+  }
+};
+
+class Person extends Inhabitant {
+  constructor(name, gender, saying, friends, species='human', legs=2, hands=2) {
+    super(name, gender, saying, friends,legs)
+    this.species = species;
```

Comment:
Remember one of the comments above?
This one:
> Why name, gender, saying and friends are generalized into a base class, while **species** and legs (that are also properties shared by all inhabitants) are not?

Reply:
I think, now I did it.